src/reducers/computeRenderedCols.js

```diff
@@ -0,0 +1,355 @@
+/**
```

Comment:
A copy-paste of [computeRenderedRows.js](https://github.com/schrodinger/fixed-data-table-2/blob/v2.0-beta/src/reducers/computeRenderedRows.js) but for columns.

*Recap:*
Once we calculate the scroll anchor, we can calculate the list of columns in the visible viewport.
We do this by iteratively filling up columns starting from the first/last column until the columns fill up the viewport.
Along the way, we record the positions and offsets of these columns in our `IntegerBufferSet` and `PrefixIntervalTree` structures respectively.